cmd/rpcdaemon/commands/erigon_nodeInfo.go

```diff
@@ -0,0 +1,11 @@
+package commands
+
+import (
+	"context"
+
+	"github.com/ledgerwatch/erigon/p2p"
+)
+
+func (api *ErigonImpl) NodeInfo(ctx context.Context) ([]p2p.NodeInfo, error) {
+	return api.ethBackend.NodeInfo(ctx, 0)
```

Comment:
if 0 is a special constant meaning "no limit" let's name it correctly :)

Reply:
Done, but I'm not sure where exactly it should be defined. From my point of view, it should be defined on GRPC level, which would lead to unnecessary protocol complexity.
This constant is used [here](https://github.com/ledgerwatch/erigon/pull/3046/files#diff-9cc7046826ce92d9cc5432cf41ed4fc14bb6907da44f771e3932302c40e8dbc2R627) and [there](https://github.com/ledgerwatch/erigon/pull/3046/files#diff-fb212cec66cb8a89d0798b314c556ba55bd8c5f0385763a87fc663bc76b00b9fR10).

Reply:
or you can just make the gRPC always return you all the sentries but limit the slice already on the RPC daemon API level. a bit more info over the wire, but then you keep the spec to the API implementation specifically.

Reply:
well, we would need to edit proto-files again for this. I can do it, but are you sure about that approach?

Reply:
I think it's fine as it is right now. We can improve that in the future.